[Docs][ML][SIEM] Combines prebuilt job descriptions (#932)
elastic/stack-docs: docs/en/siem/machine-learning.asciidoc

=== Prebuilt job reference

Prebuilt jobs automatically detect file system and network anomalies on your 

> hosts. To run jobs when you are not using {beats}, you must map your data to

Reviewer: With other ML modules, you don't even see the job suggestions unless you have the appropriate data. If that's true here as well, it might help to do something like this:
```suggestion
hosts. If you are not using {beats}, you must map your data to
```

Author: In SIEM, you can always see the jobs. They are marked as unavailable. Changing to:
To enable jobs when you are not using {beats},...

> the ECS fields listed in each job description.

Reviewer: Continuation of previous suggestion...
```suggestion
ECS fields in order to see these job recommendations.
```

The following prebuilt jobs are available:

> include::../stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc[tag=siem-jobs]

Reviewer: We typically make cross-book inclusions relative to this intrinsic attribute (https://www.methods.co.nz/asciidoc/chunked/ch31.html):
```suggestion
include::{asciidoc-dir}/../stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc[tag=siem-jobs]
```

Author: @lcawl 
Can you help? I couldn't get this to resolve. Is it only true when including files from the docs repo, not when including tags.

Reviewer: You're right.  I've changed it just use an "ml-dir" attribute, in case that path changes in the future.
